lib/suspenders/app_builder.rb

```diff
@@ -531,16 +512,12 @@ def raise_on_missing_translations_in(environment)
       uncomment_lines("config/environments/#{environment}.rb", config)
     end
 
-    def override_path_for_tests
-      if ENV['TESTING']
-        support_bin = File.expand_path(File.join('..', '..', 'spec', 'fakes', 'bin'))
-        "PATH=#{support_bin}:$PATH"
-      end
+    def run_heroku(command, environment)
+      run "heroku #{command} --remote #{environment}"
```

Comment:
Should we delegate this method too, and have it defined only once? Does `set_heroku_rails_secrets` also belong in the adapter?